packages/lib/CalEventParser.ts

```diff
@@ -188,7 +191,7 @@ ${getWho(calEvent, t)}
 ${t("where")}:
 ${getLocation(calEvent)}
 ${getDescription(calEvent, t)}
-${getAdditionalNotes(calEvent, t)}
+${hasAdditionalNotes ? additionalNotes : ""}
```

Comment:
I just noticed that, there is already a [handling for empty string inside getAdditionalNotes](https://github.com/calcom/cal.com/blob/c41583175e1c8ac1ad6ff5ff702084402cc85eed/packages/lib/CalEventParser.ts#L68)

@sohamdata Can you confirm that you are able to replicate the issue and this change fixes it?

Reply:
I saw the issue, so it's also possible that I entered a space there accidentally and it happens only in that case. We can do a trim if that's the case.

Reply:
@hariombalhara you are right, it happens when space is entered. However, when I tried checking for `calEvent.additionalNotes.trim()` in the getAdditionalNotes function, the additional notes still show up for some reason, no matter the number of spaces. Can you confirm the same?

Reply:
@hariombalhara
for debugging I changed to (sorry for the gibberish, it had catch my eye in the terminal)
```
export const getAdditionalNotes = (calEvent: CalendarEvent, t: TFunction) => {
  console.log("HRERHERHERHEREHREHREHREHRERHERH", calEvent.additionalNotes, "HRERERERER");
  calEvent.additionalNotes === calEvent.additionalNotes?.trim();
  console.log("2nd HRERHERHERHEREHREHREHREHRERHERH", calEvent.additionalNotes, "2nd HRERERERER");

  if (!calEvent.additionalNotes) {
    return "";
  }
  return `
${t("additional_notes")}:
${calEvent.additionalNotes}
  `;
};
```

terminal:
![image](https://github.com/calcom/cal.com/assets/78294692/59eee101-e699-4ed7-929d-2c5fcd20af91)
![image](https://github.com/calcom/cal.com/assets/78294692/17e279eb-03b3-4b49-8669-6d2852522813)

It's not trimming for some reason